conf/default-core.yaml

```diff
@@ -3,3 +3,4 @@ core:
   minimize: false
   dist: '{{CWD}}/.dist'
   assetPath: '{{CWD}}'
+  mobileFirst: false
```

Comment:
default: true determines whats the base breakpoint so we're not hardcoding it. mobileFirst handles changing the min/max-width